### PR TITLE
feat(BLE): BLE PHY-V2 for MAX32655 and MAX32690

### PR DESCRIPTION
**Pull request by sihyung-maxim**
### Description

- Update SVD files for MAX32655/MAX32690
- Added libphy v2
- Added vendor specific HCI commands to take advantage of new features
- Refactor existing RF Test and FCC Examples  

### Checklist Before Requesting Review

- [x] PR Title follows correct guidelines.
- [x] Description of changes and all other relevant information.
- [x] (Optional) Link any related GitHub issues [using a keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)
- [x] (Optional) Provide info on any relevant functional testing/validation.  For API changes or significant features, this is not optional.

### CODE REVIEW

**Review by EricB-ADI — APPROVED** [review]

**Comment by EricB-ADI** [comment]
Hold off on merging. ME14 might be the same.

**Review by Jacob-Scheiffler — APPROVED** [review]

File: Libraries/CMSIS/Device/Maxim/MAX32655/Include/max32655.svd [hunk 1]
```diff
@@ -3822,20 +3822,20 @@
...
+       <name>LDORXEN</name>
+       <description>LDORX Enable.</description>
        <bitOffset>0</bitOffset>
        <bitWidth>1</bitWidth>
       </field>
       <field>
-       <name>LDOTXPULLD</name>
-       <description>LDOTX Pull Down.</description>
+       <name>LDORXPULLD</name>
+       <description>LDOrX Pull Down.</description>
```
Comment (EricB-ADI) [inline Libraries/CMSIS/Device/Maxim/MAX32655/Include/max32655.svd:3832]: FYI description is lowercase

**Comment by sihyung-maxim** [comment]
These changes will not be added to the latest release.

**Review by tim-magnusen — APPROVED** [review]

**Review by lorne-maxim — APPROVED** [review]

**Review by AbbyWolf-ADI — APPROVED** [review]

**Review by yc-adi — APPROVED** [review]

**Comment by EricB-ADI** [comment]
This can be closed.

**Pull request closed without merging**